### ISSUE TITLE
Calculated fuel consumption with graph

### ISSUE THREAD

**Issue by rootz159** [akhilrex/hammond#13]
I would like to request ability to (you probably have this in "More reports" in roadmap, but maybe not):

- calculate fuel consumption between fill ups
- calculate average of fuel consumption from all fill ups (or some custom interval)
- show a graph of the fuel consumption (also with ability to select custom interval)

**Comment by akhilrex** [comment]
Yes. Reports are going to be built soon. I will keep these in mind.

Thanks and Regards

Akhil Gupta
